packages/core/src/tests/factories/documents.ts

```diff
@@ -16,6 +18,13 @@ function makeRandomDocumentVersionData() {
   }
 }
 
+export async function markAsSoftDelete(documentUuid: string, tx = database) {
+  return tx
+    .update(documentVersions)
+    .set({ deletedAt: new Date() })
+    .where(eq(documentVersions.documentUuid, documentUuid))
+}
+
```

Comment:
Add a commitId, otherwise you're marking it as deleted in the whole file history. A document should not be created being deleted (?)